### DISCUSSION TITLE
To add an OA Field !

### DISCUSSION THREAD

**Discussion by pj323** [ReaVNaiL/New-Grad-2024#212] [Ideas]
Can we add another field where it will let us know if people are receiving OA's as their first round as well. Just thinking of Ideas which can help the job search. Thanks!.

**Comment by ReaVNaiL** [comment]
I think having an extra field would make the list convoluted. Instead I suggest reaching out in cscareers discord for these follow ups instead: https://discord.com/invite/cscareers
